### PR TITLE
Feat function schedule

### PR DESCRIPTION
**Pull request by eldadfux**
## What does this PR do?

Added option to schedule execution of cloud functions using a CRON syntax. Still need to figure out what to do if the worker is failing after a function has been rescheduled but not completed.

## Test Plan

Still missing tests.

## Related PRs and Issues

#307 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by eldadfux** [comment]
@TorstenDittmann you can review this PR. I'll submit another on with the backup checks we talked about.

File: app/workers/functions.php [hunk 1]
```diff
@@ -210,6 +204,47 @@ public function perform()
...
+                ResqueScheduler::enqueueAt($next, 'v1-functions', 'FunctionsV1', [
+                    'projectId' => $projectId,
+                    'functionId' => $function->getId(),
+                    'executionId' => null,
+                    'trigger' => 'schedule',
+                    'scheduleOriginal' => $function->getAttribute('schedule', ''),
+                ]);  // Async task rescheduale
+
+                Swoole\Coroutine\run(function () use ($trigger, $projectId, $executionId, $database, $function) {
+                    $this->execute($trigger, $projectId, $executionId, $database, $function);
```
Comment (TorstenDittmann) [inline app/workers/functions.php:246]: Is there no `go()` from Swoole necessary?

Comment (eldadfux) [reply app/workers/functions.php:246]: Umm... I'll give it a check

Comment (eldadfux) [reply app/workers/functions.php]: Removed co-routines for now.

**Review by TorstenDittmann — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**